stdlib/private/StdlibCollectionUnittest/CheckRangeReplaceableCollectionType.swift

```diff
@@ -862,6 +862,22 @@ self.test("\(testNamePrefix).removeSubrange()/semantics") {
   }
 }
 
+self.test("\(testNamePrefix).replaceSubrange()/closedRange/semantics") {
+  for test in removeRangeTests {
```

Comment:
Since that enum has a case with an associated type, the direct comparison won't work. I've added a helper property to its definition, since I think it might come up elsewhere.